app/controllers/api/storage.php

```diff
@@ -449,6 +449,11 @@
                 throw new Exception(Exception::STORAGE_INVALID_CONTENT_RANGE);
             }
 
+            $idValidator = new UID();
+            if (!$idValidator->isValid($request->getHeader('x-appwrite-id'))) {
```

Comment:
x-appwrite-id is already assigned to `$fileId` why are we not using that instead?
Also, in the chunked upload, the first request will not have `x-appwrite-id` header, will that not cause error to throw?

Reply:
> x-appwrite-id is already assigned to $fileId why are we not using that instead

because the error message specifically mentions that the value of `x-appwrite-id` is invalid. Using `$fileId` may throw an error for invalid value of `$fileId`. I am not sure where the value of $fileId is validated so I might be wrong.

> the first request will not have x-appwrite-id header

This should work because the default value for `getHeader` is empty string
https://github.com/utopia-php/swoole/blob/2949a1bd93f5c772fc7de974e6b9da01106c704a/src/Swoole/Request.php#L270
and the validator at 
https://github.com/utopia-php/database/blob/d821d1f18dfe83b0350ed208be3658764aec99d7/src/Database/Validator/Key.php#L7
considers empty string to be valid https://onlinephp.io/c/21d30